package.json

```diff
@@ -21,6 +21,9 @@
     "build-watch": "swc ./src -d dist -w",
     "test": "jest ./src"
   },
+  "dependencies": {
+    "js-yaml": "^4.1.0"
+  },
```

Comment:
I moved this to a dependency because it's an actual dep of a rule. That rule I forgot to enable as part of the feature flag work that was done. Once this is merged, I'll enable it.